server/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const path = require('path');
+const paths = require('../config/paths');
+
+const PORT = process.env.PORT || 3000;
+const BUILD_PATH = path.resolve(paths.build);
+const server = express();
```

Comment:
❗️
Константы (в плане то, что мы именуем через SNAKE_CASE, так-то что то, что это через `const` объявлено и сточки зрения языка неразличимо) стоит отделять пустой строкой
```suggestion
const BUILD_PATH = path.resolve(paths.build);

const server = express();
```